package.json

```diff
@@ -9,6 +9,9 @@
     "lint": "next lint",
     "test": "jest --watch"
   },
+  "engines": {
+    "node": ">=20.11.0"
```

Comment:
I am not sure we should use `>20.x` since this is still just in beta for Vercel
![image](https://github.com/jjm3x3/AmazingRaceFantasy/assets/5040219/3f9aebb3-cc05-4fac-9bf0-7633580edd2a)